src/praisonai-deploy/praisonai_deploy/main.py

```
@@ -281,45 +281,100 @@ def destroy(self, force: bool = False) -> DestroyResult:
-                    try:
-                        os.kill(int(pid), signal.SIGTERM)
-                        deleted_resources.append(f"process:{pid}")
-                    except (ProcessLookupError, ValueError):
-                        pass
-                
-                return DestroyResult(
-                    success=True,
-                    message=f"Stopped API server on port {port}",
-                    resources_deleted=deleted_resources
+            if sys.platform == "win32":
+                subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)